src/app/BUILD.gn

```diff
@@ -63,53 +63,35 @@ buildconfig_header("app_buildconfig") {
   visibility = [ ":app_config" ]
 }
 
+source_set("app_config") {
+  sources = [ "AppConfig.h" ]
+
+  deps = [ ":app_buildconfig" ]
+}
+
 source_set("revision_info") {
   sources = [
     "DataModelRevision.h",
     "InteractionModelRevision.h",
     "SpecificationVersion.h",
   ]
-}
-
-source_set("app_config") {
-  sources = [ "AppConfig.h" ]
 
-  deps = [ ":app_buildconfig" ]
+  public_deps = [ ":app_config" ]
```

Comment:
Why is `revision_info` dependent on `app_config`? Is the intention to override `CHIP_DEVICE_DATA_MODEL_REVISION`, `CHIP_DEVICE_INTERACTION_MODEL_REVISION`, or `CHIP_DEVICE_SPECIFICATION_VERSION`?

Reply:
Removed it locally; haven't pushed that changed yet.

Reply:
@tehampson (mentioning myself so that I get notified about responses in this PR)

Reply:
@tehampson pushed the update; its ready for review